backup/protractor.conf.js

```diff
@@ -0,0 +1,71 @@
+var globalTimeout = 120000;
+
+exports.config = {
+
+    seleniumServerJar: "lib/selenium-server-standalone-2.44.0.jar",
+    seleniumPort: 4444,
+    seleniumArgs: [],
+
+    multiCapabilities: [{
+    //{browserName: 'firefox'},
+    {
+        browserName: 'chrome'
+    }
+  }],
+
+suites: {
+    location: './e2e/automation/*-spec.js',
+ 
+  },
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// A base URL for your application under test. Calls to protractor.get() with relative paths will be prepended with this. //
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//baseUrl: 'https://infoplustest.localhost-testsubdomain1.infopluswms.com:8443/',
+baseUrl: "https://gatewaydemo.qa02.infopluswms.com/",
```

Comment:
Remove these if we're not using them

Reply:
Can I remove the whole backup folder?? I wasn't sure how much impact it would be on the branch hence left it there :)